/***************************************************************************
 *   Copyright (C) 2014-2015 by Savoir-Faire Linux                         *
 *   Author : Emmanuel Lepage Vallee <elv1313@gmail.com>                   *
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; either version 3 of the License, or     *
 *   (at your option) any later version.                                   *
 *                                                                         *
 *   This program is distributed in the hope that it will be useful,       *
 *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
 *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
 *   GNU General Public License for more details.                          *
 *                                                                         *
 *   You should have received a copy of the GNU General Public License     *
 *   along with this program.  If not, see <http://www.gnu.org/licenses/>. *
 **************************************************************************/
#ifndef SECURITY_PAGE_H
#define SECURITY_PAGE_H

#include <QWidget>
#include "pagebase.h"
#include "ui_security.h"

namespace Pages {

class Security final : public PageBase, public Ui_SecurityPage
{
Q_OBJECT
public:
   //Constructor
   explicit Security(QWidget *parent = nullptr);

};

}

#endif

// kate: space-indent on; indent-width 3; replace-tabs on;
